rfcs/react-components/non-core-components.md

```diff
@@ -0,0 +1,322 @@
+# RFC: Support for components in the Fluent UI umbrella that are not core
+
+[@ling1726](https://github.com/ling1726)
+
+## Summary
```

Comment:
It would be great to have one paragraph **Motivation** chapter at the beginning.

Reply:
That's essentially the _**Summary**_  section, I don't want to deviate too much from our template